### ISSUE TITLE
Object.getClass is deprecated.

### ISSUE THREAD

**Issue by rkatsala** [t9md/atom-vim-mode-plus-move-selected-text#8]
`Base` will be not available soon, Use `service.getClass()` instead of `service.Base.getClass()`
```
Object.getClass (/home/roma/.atom/packages/vim-mode-plus/lib/main.js:227:16)
loadVmpCommands (/home/roma/.atom/packages/vim-mode-plus-move-selected-text/lib/move-selected-text.js:19:24)
Object.consumeVim (/home/roma/.atom/packages/vim-mode-plus-move-selected-text/lib/main.js:43:69)
Provider.provide (/usr/lib/atom/node_modules/service-hub/lib/provider.js:34:52)
ServiceHub.consume (/usr/lib/atom/node_modules/service-hub/lib/service-hub.js:50:18)
Package.activateServices (/usr/lib/atom/src/package.js:400:73)
```

**Comment by t9md** [comment]
Are you using latest version of this pkg?

**Comment by rkatsala** [comment]
Pkg version 0.11.1, Atom version 1.24.0.

**Comment by t9md** [comment]
The #8 was fix for this issue, but I just have forgotten to release it.
Just released v0.12.0.
Thanks for reporting this issue!
